Make sure that folders are created *before* any objects are built.

--- epstool-3.09/src/common.mak.orig	2015-03-15 11:25:28.000000000 +0100
+++ epstool-3.09/src/common.mak	2021-09-10 11:20:14.430895700 +0200
@@ -80,114 +80,114 @@
 all: $(BEGIN) $(TARGET)
 
-$(OD)calloc$(OBJ): $(SRC)calloc.c $(common_h)
+$(OD)calloc$(OBJ): $(SRC)calloc.c $(common_h) $(OD)lib.rsp
 	$(COMP) $(FOO)calloc$(OBJ) $(CO) $(SRC)calloc.c
 
 $(OD)capp$(OBJ): $(SRC)capp.c $(common_h) $(dscparse_h) $(copt_h) \
  $(capp_h) $(cdll_h) $(cgssrv_h) $(cimg_h) $(cpagec_h) $(cprofile_h) \
- $(cres_h)
+ $(cres_h) $(OD)lib.rsp
 	$(COMP) $(FOO)capp$(OBJ) $(CO) $(SRC)capp.c
 
 $(OD)cargs$(OBJ): $(SRC)cargs.c $(common_h) $(dscparse_h) $(capp_h) \
- $(cargs_h) $(copt_h) $(cview_h)
+ $(cargs_h) $(copt_h) $(cview_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cargs$(OBJ) $(CO) $(SRC)cargs.c
 
-$(OD)cbmp$(OBJ): $(SRC)cbmp.c $(common_h) $(gdevdsp_h) $(cimg_h)
+$(OD)cbmp$(OBJ): $(SRC)cbmp.c $(common_h) $(gdevdsp_h) $(cimg_h) $(OD)lib.rsp
 	$(COMP) $(LIBPNGINC) $(FOO)cbmp$(OBJ) $(CO) $(SRC)cbmp.c
 
-$(OD)ccoord$(OBJ): $(SRC)ccoord.c $(common_h) $(gdevdsp_h) $(cpagec_h)
+$(OD)ccoord$(OBJ): $(SRC)ccoord.c $(common_h) $(gdevdsp_h) $(cpagec_h) $(OD)lib.rsp
 	$(COMP) $(FOO)ccoord$(OBJ) $(CO) $(SRC)ccoord.c
 
 $(OD)cdisplay$(OBJ): $(SRC)cdisplay.c $(common_h) $(gdevdsp_h) $(errors_h) \
- $(capp_h) $(cimg_h) $(cdisplay_h) $(cgssrv_h) $(cmsg_h)
+ $(capp_h) $(cimg_h) $(cdisplay_h) $(cgssrv_h) $(cmsg_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cdisplay$(OBJ) $(CO) $(SRC)cdisplay.c
 
 $(OD)cdoc$(OBJ): $(SRC)cdoc.c $(common_h) $(dscparse_h) $(capp_h) \
- $(cdoc_h) $(cpdfscan_h) $(cres_h)
+ $(cdoc_h) $(cpdfscan_h) $(cres_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cdoc$(OBJ) $(CO) $(SRC)cdoc.c
 
 $(OD)ceps$(OBJ): $(SRC)ceps.c $(common_h) $(dscparse_h) \
  $(gdevdsp_h) $(capp_h) $(cbmp_h) $(cdoc_h) $(ceps_h) $(cimg_h) \
- $(cmac_h) $(cpdfscan_h) $(cps_h)
+ $(cmac_h) $(cpdfscan_h) $(cps_h) $(OD)lib.rsp
 	$(COMP) $(FOO)ceps$(OBJ) $(CO) $(SRC)ceps.c
 
-$(OD)cfile$(OBJ): $(SRC)cfile.c $(SRC)cfile.h
+$(OD)cfile$(OBJ): $(SRC)cfile.c $(SRC)cfile.h $(OD)lib.rsp
 	$(COMP) $(FOO)cfile$(OBJ) $(CO) $(SRC)cfile.c
 
-$(OD)clfile$(OBJ): $(SRC)clfile.c $(SRC)cfile.h
+$(OD)clfile$(OBJ): $(SRC)clfile.c $(SRC)cfile.h $(OD)lib.rsp
 	$(COMP) $(FOO)clfile$(OBJ) $(CO) $(SRC)clfile.c
 
 $(OD)cgsdll$(OBJ): $(SRC)cgsdll.c $(common_h) $(errors_h) $(iapi_h) \
- $(capp_h) $(cdll_h) $(cgsdll_h)
+ $(capp_h) $(cdll_h) $(cgsdll_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cgsdll$(OBJ) $(CO) $(SRC)cgsdll.c
 
 $(OD)cgssrv$(OBJ): $(SRC)cgssrv.c $(common_h) $(dscparse_h) \
  $(errors_h) $(iapi_h) \
  $(capp_h) $(cdll_h) $(cimg_h) $(copt_h) $(cdisplay_h) $(cdoc_h) \
- $(cgsdll_h) $(cmsg_h) $(cpdf_h) $(cpdfscan_h) $(cview_h) $(cgssrv_h)
+ $(cgsdll_h) $(cmsg_h) $(cpdf_h) $(cpdfscan_h) $(cview_h) $(cgssrv_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cgssrv$(OBJ) $(CO) $(SRC)cgssrv.c
 
-$(OD)chist$(OBJ): $(SRC)chist.c $(common_h) $(chist_h)
+$(OD)chist$(OBJ): $(SRC)chist.c $(common_h) $(chist_h) $(OD)lib.rsp
 	$(COMP) $(FOO)chist$(OBJ) $(CO) $(SRC)chist.c
 
-$(OD)cimg$(OBJ): $(SRC)cimg.c $(common_h) $(gdevdsp_h) $(cimg_h) $(clzw_h)
+$(OD)cimg$(OBJ): $(SRC)cimg.c $(common_h) $(gdevdsp_h) $(cimg_h) $(clzw_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cimg$(OBJ) $(CO) $(SRC)cimg.c
 
-$(OD)clzw$(OBJ): $(SRC)clzw.c $(clzw_h)
+$(OD)clzw$(OBJ): $(SRC)clzw.c $(clzw_h) $(OD)lib.rsp
 	$(COMP) $(FOO)clzw$(OBJ) $(CO) $(SRC)clzw.c
 
-$(OD)cmac$(OBJ): $(SRC)cmac.c $(cmac_h) $(common_h)
+$(OD)cmac$(OBJ): $(SRC)cmac.c $(cmac_h) $(common_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cmac$(OBJ) $(CO) $(SRC)cmac.c
 
-$(OD)cmbcs$(OBJ): $(SRC)cmbcs.c $(common_h)
+$(OD)cmbcs$(OBJ): $(SRC)cmbcs.c $(common_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cmbcs$(OBJ) $(CO) $(SRC)cmbcs.c
 
-$(OD)copt$(OBJ): $(SRC)copt.c $(common_h) $(dscparse_h) $(copt_h) $(cprofile_h)
+$(OD)copt$(OBJ): $(SRC)copt.c $(common_h) $(dscparse_h) $(copt_h) $(cprofile_h) $(OD)lib.rsp
 	$(COMP) $(FOO)copt$(OBJ) $(CO) $(SRC)copt.c
 
 $(OD)cpagec$(OBJ): $(SRC)cpagec.c $(common_h) $(dscparse_h) \
- $(capp_h) $(cgssrv_h) $(cimg_h) $(cpagec_h) $(cpdf_h) $(cview_h)
+ $(capp_h) $(cgssrv_h) $(cimg_h) $(cpagec_h) $(cpdf_h) $(cview_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cpagec$(OBJ) $(CO) $(SRC)cpagec.c
 
 $(OD)cpdf$(OBJ): $(SRC)cpdf.c $(common_h) $(dscparse_h) \
- $(capp_h) $(cpdf_h) $(cgssrv_h) $(cview_h)
+ $(capp_h) $(cpdf_h) $(cgssrv_h) $(cview_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cpdf$(OBJ) $(CO) $(SRC)cpdf.c
 
 $(OD)cpdfscan$(OBJ): $(SRC)cpdfscan.c $(common_h) $(dscparse_h) \
- $(capp_h) $(cpdfscan_h)
+ $(capp_h) $(cpdfscan_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cpdfscan$(OBJ) $(CO) $(SRC)cpdfscan.c
 
 $(OD)cpldll$(OBJ): $(SRC)cpldll.c $(common_h) $(errors_h) $(iapi_h) \
- $(capp_h) $(cdll_h) $(cpldll_h)
+ $(capp_h) $(cdll_h) $(cpldll_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cpldll$(OBJ) $(CO) $(SRC)cpldll.c
 
 $(OD)cplsrv$(OBJ): $(SRC)cplsrv.c $(common_h) $(dscparse_h) \
  $(errors_h) $(iapi_h) \
  $(capp_h) $(cdll_h) $(cimg_h) $(copt_h) $(cdisplay_h) $(cdoc_h) \
- $(cgsdll_h) $(cmsg_h) $(cpdf_h) $(cpdfscan_h) $(cview_h) $(cgssrv_h)
+ $(cgsdll_h) $(cmsg_h) $(cpdf_h) $(cpdfscan_h) $(cview_h) $(cgssrv_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cplsrv$(OBJ) $(CO) $(SRC)cplsrv.c
 
-$(OD)cprofile$(OBJ): $(SRC)cprofile.c $(cplat_h) $(cprofile_h)
+$(OD)cprofile$(OBJ): $(SRC)cprofile.c $(cplat_h) $(cprofile_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cprofile$(OBJ) $(CO) $(SRC)cprofile.c
 
 $(OD)cps$(OBJ): $(SRC)cps.c $(common_h) $(dscparse_h) \
- $(capp_h) $(cdoc_h)
+ $(capp_h) $(cdoc_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cps$(OBJ) $(CO) $(SRC)cps.c
 
 $(OD)cvcmd$(OBJ): $(SRC)cvcmd.c $(common_h) $(dscparse_h) \
  $(copt_h) $(capp_h) $(cdoc_h) $(cgssrv_h) $(chist_h) \
- $(cmsg_h) $(cpdfscan_h) $(cps_h) $(cres_h) $(cvcmd_h) $(cview_h)
+ $(cmsg_h) $(cpdfscan_h) $(cps_h) $(cres_h) $(cvcmd_h) $(cview_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cvcmd$(OBJ) $(CO) $(SRC)cvcmd.c
 
 $(OD)cview$(OBJ): $(SRC)cview.c $(common_h) $(dscparse_h) \
  $(capp_h) $(cdoc_h) $(cgssrv_h) $(chist_h) $(cimg_h) \
  $(cmsg_h) $(copt_h) $(cpagec_h) $(cpdf_h) $(cpdfscan_h) $(cres_h) \
- $(cvcmd_h) $(cview_h)
+ $(cvcmd_h) $(cview_h) $(OD)lib.rsp
 	$(COMP) $(FOO)cview$(OBJ) $(CO) $(SRC)cview.c
 
-$(OD)dscparse$(OBJ): $(SRC)dscparse.c $(dscparse_h)
+$(OD)dscparse$(OBJ): $(SRC)dscparse.c $(dscparse_h) $(OD)lib.rsp
 	$(COMP) $(FOO)dscparse$(OBJ) $(CO) $(SRC)dscparse.c
 
-$(OD)dscutil$(OBJ): $(SRC)dscutil.c $(dscparse_h)
+$(OD)dscutil$(OBJ): $(SRC)dscutil.c $(dscparse_h) $(OD)lib.rsp
 	$(COMP) $(FOO)dscutil$(OBJ) $(CO) $(SRC)dscutil.c
 
 $(BD)epstool$(EXE): $(OD)lib.rsp $(EPSOBJS)
@@ -196,7 +196,7 @@
 $(OD)epstool$(OBJ): $(SRC)epstool.c $(SRC)common.mak \
  $(common_h) $(copt_h) $(capp_h) $(cbmp_h) $(cdoc_h) $(cdll_h) \
  $(ceps_h) $(cimg_h) $(cmac_h) $(cps_h) $(cres_h) \
- $(dscparse_h) $(errors_h) $(iapi_h) $(gdevdsp_h)
+ $(dscparse_h) $(errors_h) $(iapi_h) $(gdevdsp_h) $(OD)lib.rsp
 	$(COMP) $(FOO)epstool$(OBJ) $(CO) -DEPSTOOL_VERSION="$(EPSTOOL_VERSION)" -DEPSTOOL_DATE="$(EPSTOOL_DATE)" $(SRC)epstool.c
 
 $(BD)epstest$(EXE): $(OD)lib.rsp $(EPSTESTOBJS)
@@ -215,10 +215,10 @@
 
 # X11/gtk+ specific
 
-$(OD)xdll$(OBJ): $(SRC)xdll.c $(common_h) $(cdll_h)
+$(OD)xdll$(OBJ): $(SRC)xdll.c $(common_h) $(cdll_h) $(OD)lib.rsp
 	$(COMP) $(FOO)xdll$(OBJ) $(CO) $(SRC)xdll.c
 
-$(OD)xnodll$(OBJ): $(SRC)xnodll.c $(common_h) $(cdll_h)
+$(OD)xnodll$(OBJ): $(SRC)xnodll.c $(common_h) $(cdll_h) $(OD)lib.rsp
 	$(COMP) $(FOO)xnodll$(OBJ) $(CO) $(SRC)xnodll.c
 
 
